about.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+
+	<head>
+		<meta charset="utf-8">
+		<meta name="description" content="">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>About coding.space</title>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+		<link rel="stylesheet" href="about.css">
+		<link rel="icon" type="image/x-icon" href="./images/favicon.ico">
+   </head>
+
+   <body>
+        
+        <div class="container black-opaque main">
+        	<div class="row">
+        		<div class="col-sm-1"></div>
+        		<div class="col-sm-9" id="header">
+        			<h1>About coding.space</h1>
+        			<a id="back" href="./index.html">back to coding.space</a>
+        		</div>
+        		<div class="col-sm-2">
+        		</div>
+        	</div>
+        	<div class="row">
+        		<div class="col-sm-1"></div>
+        		<div class="col-sm-9">
+        			<p>We believe the best way to learn to code is by coding. This website is a collection of guided projects to get you started.</p>
```

Comment:
Can you put appropriate headers before each of these sections to break them up a bit and give this a more organized feel?